draft-ietf-cdni-request-routing-extensions.xml

```diff
@@ -407,21 +407,10 @@
 
 
     <section anchor="cdni-metadata-footprint-types" title="CDNI Metadata Footprint Types">
-      <t>The CDNI Footprint Object is defined in the CDNI Metadata interface document, see section 4.2.2.2 of
-          <xref target="RFC8006"/>. The object is composed of a footprint-type, referring to some of the simple data-types
-          defined in section 4.3 of <xref target="RFC8006"/>, as well as an array of properties from the specified type.
-          Additionally, in order to facilitate future specification of other optional "footprint-types",
-          a clear process for specifying optional footprint types in an IANA registry is also specified
-          (see section 7.2 of <xref target="RFC8006"/>).
+      <t>This section specifies an extension to the CDNI Metadata footprint types defined in 
```

Comment:
I think that at this wrapper section is redundant and you can move the text here directly to the sub-section below, possibly making it a bit shorter to avoid repetition. Sorry for not being clear about this earlier.

Reply:
I hope it is better now.

Reply:
yep, short and concise